### ISSUE TITLE
CW region issue

### ISSUE THREAD

**Issue by SMamula** [lucagrulla/cw#54]
**Describe the bug**
 On ubuntu (18.04) server I have installed aws cli and it works without issue, but with cw I have region issue. When try to run cw ls groups getting this error: **could not find region configuration**. Also I tried run it like this cw ls groups --region="us-east-2", but in that case getting this error  **no valid providers in chain. Deprecated.** 

**To Reproduce**
1. install cw to ubuntu 18.04 server 
2. cw ls groups --region="us-east-2" 
3. cw ls groups

**Expected behavior**
I expect to get output like this, what works fine on my local Mac machine 
/aws/rds/cluster/item-importer-sneki-test-cluster/error
/aws/rds/cluster/snekitest-cluster/error

**Screenshots**
If applicable, add screenshots to help explain your 

- [ ] problem.
![Screenshot 2019-07-19_18-35-20-067](https://user-images.githubusercontent.com/12687413/61551216-dcce6780-aa54-11e9-9695-c9c3290e9a90.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04

**Comment by lucagrulla** [comment]
Hi,
Thanks for submitting this issue.

Do you also keep credentials and config files on the server?
https://github.com/lucagrulla/cw#aws-credentials-and-configuration

**Comment by SMamula** [comment]
Hi, 

yes I have setup it on server as well and all aws cli commands works me fine. 

Best regards

**Comment by lucagrulla** [comment]
The first error (no region in the configuration" suggests me that the files are not found. 
Can you try to be explicit from the command line not just about the region but also about the profile?

**Comment by SMamula** [comment]
I tried run it with profile, exactly with this command **cw ls groups --profile staging**
and getting the same error as before.

**Comment by okelet** [comment]
I have the same problem. I don't know much about snaps, but perhaps you need a "connection" to home, so aws library can read credentials:

```bash
$ snap connections teams-for-linux | grep home
home                      teams-for-linux:home                    :home                            -
$ snap connections cw-sh 
$
```

**Comment by lucagrulla** [comment]
ok. I do need to run some tests on Ubuntu before commenting more.

**Comment by okelet** [comment]
This is strange, calling cw snap command from installation dir directly works fine, but through snap doesn't; perhaps this makes sense about snap connections:

```bash
$ /snap/cw-sh/current/cw tail /aws/lambda/xxxxx --start 1h | tail
REPORT RequestId: b00246e8-1bdc-58e0-9402-1358ec96b01a  Init Duration: 202.43 ms        Duration: 3084.18 ms    Billed Duration: 3300 ms        Memory Size: 3008 MB    Max Memory Used: 89 MB

$ cw tail /aws/lambda/compass_massexport_pro --start 1h
no valid providers in chain. Deprecated.
        For verbose messaging see aws.Config.CredentialsChainVerboseErrors

$ which cw
/snap/bin/cw

$ ls -l /snap/bin/cw
lrwxrwxrwx 1 root root 5 jul 17 13:34 /snap/bin/cw -> cw-sh

$ which cw-sh
/snap/bin/cw-sh

$ which /snap/bin/cw-sh
/snap/bin/cw-sh
```

**Comment by lucagrulla** [comment]
I think I found the issue and it is indeed related to interfaces/connections/etc.
I'll fix it as soon as possible.

**Comment by lucagrulla** [comment]
I fixed the problem but I have not released yet; I just want some more testing (just in case).

Expect v3.2.1 over the coming days.

**Comment by lucagrulla** [comment]
I created #58 to better capture the problem.

**Comment by lucagrulla** [comment]
Fixed with v3.2.1.
